lib/label.ml

```diff
@@ -74,6 +74,8 @@ type non_det = Nd_output | Nd_command
 
 let default_non_det = Nd_output
 
+type block_kind = Raw | OCaml | Cram | Toplevel | Include
```

Comment:
My impression is that Raw is unecessary as it has no value to mdx, it simply won't be interpreted. I think the skip label already covers that case.

Reply:
I didn't think of this, if using `skip` fixes #164 I guess this PR does not have any purpose for now until we make the block type label mandatory